Makefile

```
@@ -82,20 +82,19 @@ env-update:
 
 # === Pull Request Creation ===
 
-pr-create-core:
+pr:
```

checkmake (0.2.2)

[warning] Target body for "pr" exceeds allowed length of 5 (7).

(maxbodylength)